site/content/integrate/apps/bindings/_index.md

```diff
@@ -62,21 +64,25 @@ For commands we can distinguish between leaf commands (executable subcommand) an
+| `description` | string   | (Optional) Description line on command autocomplete.                                                                      |
+| `bindings`    | Bindings | List of subcommands.                                                                                                      |
+| `call`        | Call     | (Optional) Call to be inherited by all its subcommands.                                                                   |
+| `form`        | Form     | (Optional) Form to be inherited by all its subcommands.                                                                   |
 
 A leaf command must include:
 
 | Name          | Type   | Description                                                                                                                                  |
 | :------------ | :----- | :------------------------------------------------------------------------------------------------------------------------------------------- |
-| `location`    | string | The label to use to define the command.                                                                                                      |
-| `hint`        | string | (Optional) Hint line on command autocomplete.                                                                                                |
-| `description` | string | (Optional) Description line on command autocomplete.                                                                                         |
-| `call`        | Call   | Call to perform when executing the command.                                                                                                  |
+| `location`    | string | Name of this location. The whole path of locations will be added in the context. Must be unique in its level.                                   |
+| `label`       | string | The label to use to define the command. Cannot include spaces or tabs. Defaults to location. Must be unique in its level.                       |
+| `hint`        | string | (Optional) Hint line on command autocomplete.                                                                                                   |
+| `description` | string | (Optional) Description line on command autocomplete.                                                                                           |
+| `call`        | Call   | (Optional) Call to perform when executing the command. You must provide a Call if there is no Form, or the Form itself does not have a Call. |                                                                                                    |
 | `form`        | Form   | (Optional) Form representing the parameters the command can receive. If no form is provided, a form call will be made to the specified call. |
 
 The context of the call for these bindings will include the user ID, the post ID, the root post ID (if any), the channel ID and the team ID. It will also include the raw command.
```

Comment:
```suggestion
The context of the call for these bindings will include the user ID, the post ID, the root post ID (if any), the channel ID, and the team ID. It will also include the raw command.
```